adafruit_irremote.py

```diff
@@ -341,14 +346,14 @@ class GenericTransmit:
     :param bool debug: Enable debug output, default False
     """
 
-    def __init__(self, header, one, zero, trail, *, debug=False):
+    def __init__(self, header, one, zero, trail, *, debug=False) -> None:
         self.header = header
         self.one = one
         self.zero = zero
         self.trail = trail
         self.debug = debug
 
-    def transmit(self, pulseout, data, *, repeat=0, delay=0, nbits=None):
+    def transmit(self, pulseout, data, *, repeat=0, delay=0, nbits=None) -> None:
```

Comment:
`pulseout`, `data`, `repeat`, `delay`, and `nbits` arguments in this function can be typed.